apps/dashboard/src/app/__snapshots__/App.test.tsx.snap

```diff
@@ -12176,6 +12176,7 @@ exports[`FairnessWizardV1 should render successfully 1`] = `
               "native-country",
               "income",
             ],
+            "id": "abc",
```

Comment:
In a real scenario, will this still be a string like "abc" or rather a number or something entirely different?

Reply:
it will be a guid provided by SDK. Please check description.